activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -158,7 +158,9 @@ def travel_to(date_or_time)
       end
 
       # Returns the current time back to its original state, by removing the stubs added by
-      # +travel+ and +travel_to+.
+      # +travel+, +travel_to+ and +freeze_time+.
+      #
+      # +travel_back+ is also aliased as +unfreeze_time+.
```

Comment:
Don't need to document this, our docs generator will automatically add it

Reply:
That's good to know, this line is now removed from `time_helpers.rb`